src/megaclient.cpp

```diff
@@ -10716,6 +10725,9 @@ void MegaClient::fetchnodes(bool nocache)
         if (!loggedinfolderlink())
         {
             reqs.add(new CommandGetUA(this, uid.c_str(), ATTR_DISABLE_VERSIONS, NULL, 0));
+
+            static AccountDetails ad_useless;
+            reqs.add(new CommandGetUserQuota(this, &ad_useless, false, false, true, 99));
```

Comment:
We should eventually document the `source`s in use. Here the `99` seems to be a magic number, I guess it doesn't collide with the sources used by MEGAsync?

Reply:
I think 99 is fine for now, we should remove that parameter from the function anyway, later after the uq issues are solved.   However, this GetUserQuota is doubling up on the ones sent by the apps such as MEGAsync anyway, perhaps the app should ask the SDK to start the timer?  Or the SDK could just listen in on the replies to requests the app is sending anyway?

Reply:
You are correct, the parameter should eventually be removed. Forget about adding documentation about it ;)

As for the double calls, from SDK and app, the call from the app based on a timer would be removed in pro of a more low-level call from the SDK, and the SDK would notify all the apps as needed.